### PR TITLE
Feature: ireland support

### LINKED ISSUE #128: Add support for Ireland
I'd like to see Ireland as a supported country on this website. 

We can also add references to CATU - Ireland's tenant union.

### PR DESCRIPTION
**Pull request by DoxxableSetanta**
- Add resources for Ireland support 
- Hiding post code for Ireland
- Readme changes

closes #128

### CODE REVIEW

**Comment by DoxxableSetanta** [comment]
If ok with code - please hold off on merging until we discuss release strategy

File: components/modal/EditReviewModal.tsx
Comment (kellenwiltshire) [inline components/modal/EditReviewModal.tsx:1]: Small change for this file, we should remove the postal code box for Ireland to be consistent.

image

**Review by kellenwiltshire — CHANGES_REQUESTED** [review]
Small change needed, otherwise looks good!

File: components/modal/EditReviewModal.tsx
Comment (DoxxableSetanta) [reply components/modal/EditReviewModal.tsx:1]: Cool, thanks! Should have bandwith later in week to address this

**Review by kellenwiltshire — APPROVED** [review]
Looks good! Merging into the develop branch.

**Pull request merged**